metadata/src/main/java/org/apache/kafka/controller/QuorumController.java

```diff
@@ -1142,15 +1142,19 @@ private void renounce() {
             purgatory.failAll(newNotControllerException());
 
             if (snapshotRegistry.hasSnapshot(lastCommittedOffset)) {
-                newBytesSinceLastSnapshot = 0;
                 snapshotRegistry.revertToSnapshot(lastCommittedOffset);
                 authorizer.ifPresent(a -> a.loadSnapshot(aclControlManager.idToAcl()));
             } else {
-                resetState();
+                log.warn("Unable to find last committed offset {} in snapshot registry; resetting " +
```

Comment:
Is not having a snapshot for the committed offset an expected or unexpected state?

Reply:
it is expected, unfortunately. we have some ideas for fixing this, but it's outside the scope of this PR.

I guess you're hinting that this should be an INFO? :) I'll make it an INFO.